osquery/tests/integration/tables/time.cpp

```diff
@@ -16,36 +15,30 @@
 
 namespace osquery {
 
-class time : public IntegrationTableTest {};
-
-TEST_F(time, test_sanity) {
-  // 1. Query data
-  // QueryData data = execute_query("select * from time");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See IntegrationTableTest.cpp for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidatatioMap row_map = {
-  //      {"weekday", NormalType}
-  //      {"year", IntType}
-  //      {"month", IntType}
-  //      {"day", IntType}
-  //      {"hour", IntType}
-  //      {"minutes", IntType}
-  //      {"seconds", IntType}
-  //      {"timezone", NormalType}
-  //      {"local_time", IntType}
-  //      {"local_timezone", NormalType}
-  //      {"unix_time", IntType}
-  //      {"timestamp", NormalType}
-  //      {"datetime", NormalType}
-  //      {"iso_8601", NormalType}
-  //}
-  // 4. Perform validation
-  // EXPECT_TRUE(validate_rows(data, row_map));
+class Time : public IntegrationTableTest {};
+
+TEST_F(Time, test_sanity) {
+  QueryData data = execute_query("select * from time");
+
+  ASSERT_EQ(data.size(), 1ul);
+
+  ValidatatioMap row_map = {
+      {"weekday", NonEmptyString},
+      {"year", IntType},
+      {"month", IntType},
+      {"day", IntType},
+      {"hour", IntType},
+      {"minutes", IntType},
+      {"seconds", IntType},
+      {"timezone", NonEmptyString},
+      {"local_time", IntType},
+      {"local_timezone", NonEmptyString},
+      {"unix_time", IntType},
+      {"timestamp", NonEmptyString},
```

Comment:
is timestamp a number?

Reply:
No, timestamp is something like `Wed Aug 29 16:58:23 2018 UTC`.